RF Feed - fixed fetch indicators json exception (#11830)
demisto/content: Packs/FeedRecordedFuture/ReleaseNotes/1_0_8.md

#### Integrations
##### Recorded Future RiskList Feed

> - Fixed an issue where **fetch indicators** failed due to missing EvidenceDetails value in the received indicator.

Reviewer: ```suggestion
- Fixed an issue where ***fetch-indicators*** would fail when the received indicator was missing the *EvidenceDetails* value.
```
